test/test_game.py

```diff
@@ -0,0 +1,221 @@
+import unittest
+
+from rock_paper_scissors.core.exc import InvalidRoundsError, InvalidChoiceError
+from rock_paper_scissors.core.game import Game
+from rock_paper_scissors.core.moves import Paper, Rock
+from rock_paper_scissors.core.user import User
+from rock_paper_scissors.core.valid_choices import VALID_CHOICES
+
+
+class GameInitTestCase(unittest.TestCase):
+    def test_player1_and_player2_and_starting_rounds_types(self):
+        player1 = 'Joe'
+        player2 = 'Jane'
+        game = Game(player1, player2)
+
+        self.assertIsInstance(game.player1, str)
+        self.assertIsInstance(game.player2, str)
+        self.assertIsInstance(game._rounds, int)
+        self.assertEqual(game._rounds, 0)
+
+
+class GameSetRoundsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.game = Game('player1', 'player2')
+
+    def test_valid_rounds_input(self):
+        value = self.game.rounds = 11
+
+        self.assertIsInstance(value, int)
+        self.assertGreater(value, 0)
+
+    def test_raises_error_if_round_not_an_int(self):
+        with self.assertRaises(InvalidRoundsError):
+            self.game.rounds = 'a'
```

Comment:
It's also a good idea to test whether the exception was raised with the expected error message. Fun fact: the `with self.assertRaises` context manager returns a variable with a `.exception` attribute which contains the exception that was raised within the context manager.

You can use it like this:
```python
with self.assertRaises(InvalidRoundsError) as ctx:
    self.game.rounds = 'a'

self.assertEqual(
    str(ctx.exception),
    'Choice must be an integer'
)
```